libs/sdk-ui-ext/src/internal/components/pluggableVisualizations/areaChart/tests/PluggableAreaChart.test.tsx

```diff
@@ -120,6 +127,179 @@ describe("PluggableAreaChart", () => {
+                    {
+                        buckets: [
+                            referencePointMocks.threeDatesInColumnChart.buckets[0],
+                            {
+                                localIdentifier: "view",
+                                items: referencePointMocks.threeDatesInColumnChart.buckets[1].items.slice(
+                                    0,
+                                    1,
+                                ),
+                            },
+                            {
+                                localIdentifier: "stack",
+                                items: referencePointMocks.threeDatesInColumnChart.buckets[1].items.slice(
+                                    1,
+                                    2,
+                                ),
+                            },
+                        ],
+                    },
+                ],
+                [
+                    "from column to area chart: first attribute is not date (date should get preference)",
+                    referencePointMocks.multipleDatesNotAsFirstReferencePointWithSingleMeasureColumn,
+                    {
+                        buckets: [
+                            referencePointMocks.multipleDatesNotAsFirstReferencePointWithSingleMeasureColumn
+                                .buckets[0],
+                            {
+                                localIdentifier: "view",
+                                items: [
+                                    referencePointMocks
+                                        .multipleDatesNotAsFirstReferencePointWithSingleMeasureColumn
+                                        .buckets[1].items[1],
+                                ],
+                            },
+                            {
+                                localIdentifier: "stack",
+                                items: referencePointMocks
+                                    .multipleDatesNotAsFirstReferencePointWithSingleMeasureColumn.buckets[2]
+                                    .items,
+                            },
+                        ],
+                    },
+                ],
+            ];
+            it.each(inputs)(
+                "should return correct extended reference (%s)",
+                async (
+                    _description,
+                    inputReferencePoint: IReferencePoint,
+                    expectedReferencePoint: Partial<IExtendedReferencePoint>,
+                ) => {
+                    const areaChart = createComponent({
+                        ...defaultProps,
+                        featureFlags: {
+                            enableMultipleDatesDEV: true,
+                        },
+                    });
+
+                    const referencePoint = await areaChart.getExtendedReferencePoint(inputReferencePoint);
+                    expect(referencePoint).toMatchObject(expectedReferencePoint);
```

Comment:
you could save a lot of mindless typing by using toMatchSnapshot. let the 'computer type the right side of the equation' => verify the snapshot

Reply:
I know, there's typing. It may be a subjective opinion, but for me, these are more readable than snapshots due to this main reason:
These can be written by hand and for every reference point, you can guess and write down how the resulting ref. point would look like and create test before actually implementing the rule and do the TDD. It helped me also with previous charts, for example with bugfixing where I could write test beforehand. I cannot do this with snapshot. Also, this test is focused on buckets only where the logic resides, not the other parts of the reference point.